components/server/src/server.rs

```diff
@@ -400,6 +423,34 @@ impl<ER: RaftEngine> TiKvServer<ER> {
         ConfigController::new(config)
     }
 
+    /// Run a TiKV server in recovery mode
+    /// recovery mode include:
+    /// 1. no election happen between raft group
+    /// 2. peer valid during a recovery time even without leader in its region
+    /// 3. PD can not put any peer into tombstone
+    /// 4. must ensure all region data with ts less than backup ts (below commit index) are safe
+    fn enter_recovery_mode(
+        config: &mut TiKvConfig,
+    ) {
+        // TOOD: if we do not have to restart TiKV, then, we need exit the recovery mode and config the following parameter back
+        // disable some configuration to ban raft election etc.
+        // For block-level recovery, no raft peers can start a new election.
+        let bt = config.raft_store.raft_base_tick_interval.0;
```

Comment:
I suggest to set `raft_base_tick_interval` to a large value, too. So that all unexpected changes can be avoid during recovering.

Reply:
keep it since the base tick used for triggered the log appended and raft basic logic